src/model/base_model.cc

```diff
@@ -139,6 +140,7 @@ void BaseModel::SetUnsynced() {
 
 void BaseModel::ClearUnsynced() {
     Unsynced.Set(false);
+    ClearValidationError();
```

Comment:
Just a small remark: `Unsynced.Set(false)` here is not necessary any more as it's present in `SetValidationError` method.

Reply:
👌